nx.json

```diff
@@ -0,0 +1,119 @@
+{
+  "targetDefaults": {
+    "build": {
+      "dependsOn": ["^build"],
+      "outputs": ["{projectRoot}/dist"],
+      "cache": true
+    },
+    "coverage": {
+      "dependsOn": ["^coverage"],
+      "outputs": ["{projectRoot}/coverage"],
+      "cache": true
+    },
+    "docs:build": {
+      "dependsOn": ["^docs:build"]
+    },
+    "test": {
+      "dependsOn": ["^test"],
+      "cache": true
+    },
+    "test:browser": {
+      "dependsOn": ["^test:browser"],
+      "cache": true
+    },
+    "test:node": {
+      "dependsOn": ["^test:node"],
+      "cache": true
+    },
+    "tsc": {
+      "dependsOn": ["^tsc"],
+      "cache": true
+    },
+    "build:browser": {
+      "dependsOn": ["^build:browser"]
+    },
+    "build:common": {
+      "dependsOn": ["^build:common"]
+    },
+    "client:start:ts": {
+      "dependsOn": ["^client:start:ts"]
+    },
+    "client:start:js": {
+      "dependsOn": ["^client:start:js"]
+    },
+    "client:start": {
+      "dependsOn": ["^client:start"]
+    },
+    "client:start:dev1": {
+      "dependsOn": ["^client:start:dev1"]
+    },
+    "client:start:dev2": {
+      "dependsOn": ["^client:start:dev2"]
+    },
+    "test:cli": {
+      "dependsOn": ["^test:cli"],
+      "cache": true
+    },
+    "test:integration": {
+      "dependsOn": ["^test:integration"],
+      "cache": true
+    },
+    "test:unit": {
+      "dependsOn": ["^test:unit"],
+      "cache": true
+    },
+    "coverage:test": {
+      "dependsOn": ["^coverage:test"],
+      "cache": true
+    },
+    "profiling": {
+      "dependsOn": ["^profiling"]
+    },
+    "benchmarks": {
+      "dependsOn": ["^benchmarks"]
+    },
+    "build:benchmarks": {
+      "dependsOn": ["^build:benchmarks"]
+    },
+    "test:API": {
+      "dependsOn": ["^test:API"]
+    },
+    "test:blockchain": {
+      "dependsOn": ["^test:blockchain"]
+    },
+    "test:blockchain:allForks": {
+      "dependsOn": ["^test:blockchain:allForks"]
+    },
+    "test:blockchain:transitionForks": {
+      "dependsOn": ["^test:blockchain:transitionForks"]
+    },
+    "test:buildIntegrity": {
+      "dependsOn": ["^test:buildIntegrity"]
+    },
+    "test:state": {
+      "dependsOn": ["^test:state"]
+    },
+    "test:state:allForks": {
+      "dependsOn": ["^test:state:allForks"]
+    },
+    "test:state:selectedForks": {
+      "dependsOn": ["^test:state:selectedForks"]
+    },
+    "test:state:slow": {
+      "dependsOn": ["^test:state:slow"]
+    },
+    "lint": {
+      "cache": true
+    },
+    "lint:diff": {
+      "cache": true
+    },
+    "lint:fix": {
+      "cache": true
+    }
+  },
+  "affected": {
+    "defaultBase": "master"
+  },
+  "nxCloudAccessToken": "NDU2NDU3NmMtZjBiYy00MzBiLTkwMjEtNzczOGZhY2NiMDUzfHJlYWQtd3JpdGU="
```

Comment:
This nx cloud token is doing nothing atm until someone claims it. It's meant to be public and a readonly token. Note: ethereumjs as an OSS project qualifies for free nx cloud access.

It doesn't require a remote cache and is optional. Without it you still benifit from your local cache

Reply:
Based on the comment about not wanting dependency I'm guessing that this repo would not want to take advantage of nx cloud and thus we would remove this configuration here
```suggestion
```

Reply:
Want to remove this because some folks using ethereumjs monorepo won't want the network requests happening that nx cloud removes. Better to just use the local only cache